### ISSUE TITLE
Add config to override the narrator speed

### ISSUE THREAD

**Issue by TeamDman** [TeamDman/SuperFactoryManager#255]
The default narrator in Minecraft does not speak fast despite configuring speed in the Windows narrator settings.
It should be possible to respect the Windows settings or to add a client config value to modify the speed.

This is overriding a Vanilla mechanic and deserves a standalone mod but practicality may just have it be a part of SFM.
